v2/go.mod

```diff
@@ -8,14 +8,13 @@ require (
 	// bump: gorilla/sessions /github\.com\/gorilla\/sessions v(.*)/ https://github.com/gorilla/sessions.git|^1
 	// bump: gorilla/sessions command cd v2 && go get -d github.com/gorilla/sessions@v$LATEST && go mod tidy
 	github.com/gorilla/sessions v1.2.1
-	github.com/mattn/go-sqlite3 v2.0.3+incompatible // indirect
 	github.com/pkg/errors v0.9.1 // indirect
 	golang.org/x/crypto v0.0.0-20211108221036-ceb1ce70b4fa // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
-	gorm.io/driver/mysql v1.3.2
-	gorm.io/driver/postgres v1.3.1
-	gorm.io/driver/sqlite v1.1.4
+	gorm.io/driver/mysql v1.2.2
+	gorm.io/driver/postgres v1.2.3
+	gorm.io/driver/sqlite v1.2.6
 	// bump: gorm.io/gorm /gorm\.io\/gorm v(.*)/ https://github.com/go-gorm/gorm.git|^1
 	// bump: gorm.io/gorm command cd v2 && go get -d gorm.io/gorm@v$LATEST && go mod tidy
-	gorm.io/gorm v1.23.4
+	gorm.io/gorm v1.22.4
```

Comment:
Should gorm.io/gorm and the drivers really be downgraded?

Reply:
The changes that were made were specific for the project that we are working on and these are the dependencies we used. When using 1.23.4 there were issues.

Reply:
The issues were coming from the upgraded version of gorm.io/gorm in the project that we are working on. However, we can use the forked repo, so you necessarily need to merge this pr. I am glad we were able to bring this to your attention.

Reply:
👍  yes much appreciated! do you want to revert the other version changes in this PR or should i do a separate PR?